lib/src/sliver_infinite_list.dart

```diff
@@ -154,6 +158,7 @@ class _SliverInfiniteListState extends State<SliverInfiniteList> {
     return SliverList(
       delegate: SliverChildBuilderDelegate(
         childCount: effectiveItemCount,
+        findChildIndexCallback: widget.findChildIndexCallback,
```

Comment:
Could you also add a test to verify that the method is being passed through correctly? We don't need to test that it works but for future's sanity sake it would be nice to have it being touched in tests

Reply:
Hi @wolfenrain, I added the test